contracts/epochs/FlowEpoch.cdc

```diff
@@ -426,7 +426,6 @@ pub contract FlowEpoch {
         pub fun resetEpoch(
             currentEpochCounter: UInt64,
             randomSource: String,
-            newPayout: UFix64?,
```

Comment:
⚠️  This is a breaking change - we'll need to update the [spork util](https://github.com/onflow/flow-go/blob/c534287f8f66653f4dd582a3aa3187d65e564b61/cmd/util/cmd/epochs/cmd/reset.go) as well. 

When we merge this, let's make sure to communicate the breaking change to spork operators.

Reply:
Yep, thanks for flagging!

Reply:
Here's the PR for the spork util: https://github.com/onflow/flow-go/pull/4286